content/doc/book/managing/plugins.adoc

```diff
@@ -225,6 +225,67 @@ The configuration(s) created by the disabled plugin behave as if the plugin
 were uninstalled, insofar that they result in warnings on boot but are
 otherwise ignored.
 
+==== Using the Jenkins CLI
+
+It is also possible to enable and disable plugins using the <<cli#,Jenkins CLI>>.
+Since https://jenkins.io/changelog/#v2.136[Jenkins 2.136], administrators may 
+invoke the `enable-plugin` command to re-enable a collection of plugins which 
+were previously disabled and since https://jenkins.io/changelog/#v2.151[Jenkins 2.151] 
```

Comment:
```suggestion
were previously disabled, and since https://jenkins.io/changelog/#v2.151[Jenkins 2.151] 
```

Reply:
I don't think that proposed comma should be there.

(There's a significant explanation of the use of commas here: https://www.grammarly.com/blog/comma/ . In this case, the "and" is linking two separate clauses so the comma should not be needed.)

Reply:
After re-reading the sentence a few times, I see what you mean.